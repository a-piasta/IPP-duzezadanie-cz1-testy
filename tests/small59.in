extendRoute;50;gzhcbqoyikexvvtptnxtsohegnuce
getRouteDescription;742
repairRoad;euqizdqfrcjoiew;fbbdyirpjyuat;-9
getRouteDescription;573
addRoad;hadpmlsfqwpc;bclgeomewdtgd;25;-11
extendRoute;50;fbbdyirpjyuat
repairRoad;bclgeomewdtgd;ekkvovlwbjkxpwenkrwsv;-6
newRoute;452;wgkgxmnhicwndjijgat;wgkgxmnhicwndjijgat
getRouteDescription;452
addRoad;iykragjtppfan;hcwsrhdjjwoeh;22;7
repairRoad;euqizdqfrcjoiew;hcwsrhdjjwoeh;-3
repairRoad;ykyfpbxdahqgenezc;hadpmlsfqwpc;9
getRouteDescription;452
removeRoad;hadpmlsfqwpc;bclgeomewdtgd
extendRoute;452;hcwsrhdjjwoeh
removeRoad;iykragjtppfan;hcwsrhdjjwoeh
addRoad;bclgeomewdtgd;hadpmlsfqwpc;9;0
extendRoute;452;euqizdqfrcjoiew
extendRoute;452;ykyfpbxdahqgenezc
getRouteDescription;452
addRoad;wgkgxmnhicwndjijgat;hcwsrhdjjwoeh;28;-13
getRouteDescription;452
addRoad;fbbdyirpjyuat;fbbdyirpjyuat;24;-14
repairRoad;fbbdyirpjyuat;euqizdqfrcjoiew;-7
newRoute;372;wgkgxmnhicwndjijgat;ekkvovlwbjkxpwenkrwsv
newRoute;920;gzhcbqoyikexvvtptnxtsohegnuce;ykyfpbxdahqgenezc
addRoad;ekkvovlwbjkxpwenkrwsv;hcwsrhdjjwoeh;27;-15
getRouteDescription;152
addRoad;fbbdyirpjyuat;ykyfpbxdahqgenezc;3;-14
repairRoad;fbbdyirpjyuat;ykyfpbxdahqgenezc;12
newRoute;946;ykyfpbxdahqgenezc;ykyfpbxdahqgenezc
removeRoad;bclgeomewdtgd;hadpmlsfqwpc
getRouteDescription;920
getRouteDescription;946
removeRoad;hadpmlsfqwpc;bclgeomewdtgd
addRoad;iykragjtppfan;iykragjtppfan;15;2
newRoute;923;hadpmlsfqwpc;ekkvovlwbjkxpwenkrwsv
removeRoad;hadpmlsfqwpc;bclgeomewdtgd
addRoad;ekkvovlwbjkxpwenkrwsv;bclgeomewdtgd;6;-4
repairRoad;wgkgxmnhicwndjijgat;hcwsrhdjjwoeh;14
repairRoad;wgkgxmnhicwndjijgat;iykragjtppfan;-12
addRoad;ykyfpbxdahqgenezc;hcwsrhdjjwoeh;3;14
removeRoad;hadpmlsfqwpc;bclgeomewdtgd
newRoute;817;euqizdqfrcjoiew;bclgeomewdtgd
getRouteDescription;736
repairRoad;wgkgxmnhicwndjijgat;euqizdqfrcjoiew;-5
repairRoad;ekkvovlwbjkxpwenkrwsv;euqizdqfrcjoiew;-15
repairRoad;ykyfpbxdahqgenezc;fbbdyirpjyuat;-8
extendRoute;923;hadpmlsfqwpc
newRoute;978;euqizdqfrcjoiew;ykyfpbxdahqgenezc
newRoute;419;ekkvovlwbjkxpwenkrwsv;bclgeomewdtgd
removeRoad;bclgeomewdtgd;ekkvovlwbjkxpwenkrwsv
addRoad;hcwsrhdjjwoeh;euqizdqfrcjoiew;8;-12
extendRoute;419;iykragjtppfan
getRouteDescription;920
getRouteDescription;978
newRoute;314;ykyfpbxdahqgenezc;euqizdqfrcjoiew
newRoute;914;euqizdqfrcjoiew;ekkvovlwbjkxpwenkrwsv
repairRoad;ekkvovlwbjkxpwenkrwsv;wgkgxmnhicwndjijgat;10
