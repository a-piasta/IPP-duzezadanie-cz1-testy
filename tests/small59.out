1: NIE
2: 
3: NIE
4: 
5: TAK
6: NIE
7: NIE
8: NIE
9: 
10: TAK
11: NIE
12: NIE
13: 
14: TAK
15: NIE
16: TAK
17: NIE
18: NIE
19: NIE
20: 
21: TAK
22: 
23: NIE
24: NIE
25: NIE
26: NIE
27: TAK
28: 
29: TAK
30: TAK
31: NIE
32: NIE
33: 
34: 
35: NIE
36: NIE
37: NIE
38: NIE
39: TAK
40: TAK
41: NIE
42: TAK
43: NIE
44: NIE
45: 
46: NIE
47: NIE
48: NIE
49: NIE
50: NIE
51: TAK
52: NIE
53: TAK
54: NIE
55: 
56: 
57: TAK
58: TAK
59: NIE
